analytics/upress_reporting/models/aggregators/aggregator.py

```diff
@@ -11,13 +12,13 @@ def __init__(self, publisher, date_range):
         self.publisher = publisher
         self.date_range = date_range
         self.s3_client = boto3.client("s3")
-
-    @abstractmethod
-    def pull_interaction_events(self) -> list[InteractionEvent]:
-        return
+        self.referrer_url = os.environ.get("REFERRER_URL", None)
 
     @abstractmethod
     def match_log_info_with_drb_data(self, log_object) -> InteractionEvent | None:
+        '''
+        Match S3 log info to data in DRB dbs.
+        '''
```

Comment:
I think your function naming is great and explicit! What are your thoughts on removing these top-level function comments?

Reply:
Fine by me! Do you want to remove these comments for every single function?

Reply:
I think so!

Reply:
Sounds good! I removed most of the comments except for the key, more-vaguely named functions (`pull_interaction_events` and `aggregate_interaction_events`).